Sources/PackageModel/Resource.swift

```diff
@@ -26,9 +26,9 @@ public struct Resource: Codable, Equatable {
     public var destination: RelativePath {
         switch self.rule {
         case .process(.some(let localization)):
-            return RelativePath("\(localization).\(Self.localizationDirectoryExtension)/\(path.basename)")
+            return try! RelativePath(validating: "\(localization).\(Self.localizationDirectoryExtension)/\(path.basename)") // try! safe
```

Comment:
Doesn't seem correct to call this "safe", `Self.localizationDirectoryExtension` could be any string.